style.css

```diff
@@ -145,9 +149,71 @@ a:hover {
     color: var(--white);
 }
 
+/* card images */
+
+.image-card__profile {
+    position: absolute;
+    left: 1em;
+    bottom: 1.5em;
```

Comment:
These styles should not be here. They should only apply if we have `hover` meaning inside the media query with the hover.